README.md

```diff
@@ -0,0 +1,38 @@
+# FakedPeopleBot
+
+This program is a group study project for ITMO University. </br> It allows to generate fake people's names, surnames and addresses. </br>
```

Comment:
Probably we could replace list of data generated by bot to `fake data about persons`? Because in this case we will not need to add new options here. This list is not full, by the way. It also generates job positions and age